### PR TITLE
Update TETTRIs Service schema

### PR DESCRIPTION
**Pull request by TomDijkema**
Update the JSON schema for the TETTRIs taxonomic service to align with the changes within in DiSSCo.
Version remains at 0.1.0.

### CODE REVIEW

**Review by samleeflang — COMMENTED** [review]
Old comments, but we discussed this and all terms need to come from schema was the conclusion. If I remember, correctly. Maybe close this PR and reopen one when all changes have been made.

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json [hunk 1]
```diff
@@ -1,211 +1,332 @@
...
+          "examples": [
+              "20.500.14547/47bcc386b52ecbae623a"
+          ]
+      },
+      "@type": {
+          "type": "string",
+          "description": "The type of the object, in this case taxonomicService",
+          "const": "taxonomicService"
+      },
+      "cetaf:created": {
```
[the remark this answers is not in the document]
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json]: We opted to use schema.org terms for the whole schema, except one ods term (topic discipline)

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json [hunk 2]
```diff
@@ -1,211 +1,332 @@
...
+          ]
+      },
+      "dcterms:modified": {
+          "type": "string",
+          "description": "Modification date for Taxonomic Service information",
+          "examples": [
+              "2023-07-10T12:02:00.000Z"
+          ]
+      },
+      "cetaf:state": {
```
[the remark this answers is not in the document]
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json]: Added enum to status

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json [hunk 3]
```diff
@@ -1,211 +1,332 @@
...
-          "type": "string",
-          "examples": [
-            "https://info.taxonomy.com"
-          ]
-        },
-        "dependencies": {
+      "@id": {
+          "type": "string",
+          "description": "The unique identifier of the Taxonomic Service object",
+          "pattern": "20.500.14547\/(.){20}",
```
[the remark this answers is not in the document]
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json]: I am not a stat at Regex, @southeo can you give me some advice on this? I already tried and updated the Regex string with something new

**Review by southeo — APPROVED** [review]
# :tada:

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (southeo) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json]: `https:\/\/hdl\.handle\.net\/20\.500\.14547\/.{20}`

[regex101](https://regex101.com/r/jpSV4p/1)

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json [hunk 4]
```diff
@@ -1,211 +1,380 @@
...
+            }
+        },
+        "schema:documentation": {
+            "type": "string",
+            "description": "URL to further documentation describing the service in more detail.",
+            "examples": [
+                "cetaf.org"
+            ]
+        },
+        "schema:ratingValue": {
```
Comment (southeo) [inline data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json:106]: Can this be negative? maybe add `"minumum":0` if not

Comment (southeo) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json:106]: https://json-schema.org/understanding-json-schema/reference/numeric#range

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json [hunk 5]
```diff
@@ -1,211 +1,380 @@
...
+                    ]
+                },
+                "schema:description": {
+                    "type": "string",
+                    "description": "A description of the Service",
+                    "examples": [
+                        "A Bombus reference collection that contains a bunch of different specimen related to the Bombus species"
+                    ]
+                },
+                "schema:slogan": {
```
Comment (southeo) [inline data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json:158]: Why is this here? It seems unnecessary

**Review by southeo — COMMENTED** [review]
:giraffe:

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json]: Thx! Almost got that in my local setup, but improved it with your example Sou

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json (hunk 4 above)
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json:106]: That's a good one, added minimum and maximum for this field

File: data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json (hunk 5 above)
Comment (TomDijkema) [reply data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json:158]: It was in the original schema of the EOSC Marketplace we based the schema on. That is why it is still in there, but it was first called 'tagLine'. We use it in the marketplace as a kind of subtitle, its up to @wouteraddink if the thinks this field is necessary or not. I think it does add some value, but not that much that we could not live without it.

**Review by southeo — APPROVED** [review]
:feather:
